### DISCUSSION TITLE
System update (forced update) for questionSet

### DISCUSSION THREAD

**Discussion by githubpiyush** [Sunbird-inQuiry/Community#86] [Q&A] [answered]
@krgauraw

How to use system update to update any questionSet?
Note :- It's a forced update which does not require versionKey.

CC :- @aks30

**Comment by krgauraw** [comment, marked as answer]
@githubpiyush Yes you can use system update api to apply the change without affecting the object state. the change will be applied to even live objects and its image node (if exists) as well together. There is no data validation in this api and doesn't require any versionKey. So Please make sure that you are sending correct data.

May i know, in which scenario, are you planning to use this api?

Hope this answer your question!

cc: @aks30 @sajeshkayyath
